src/app/clusters/ota-requestor/GenericOTARequestorDriver.h

```diff
@@ -87,7 +87,7 @@ class GenericOTARequestorDriver : public OTARequestorDriver
     uint32_t mPeriodicQueryTimeInterval = (24 * 60 * 60); // Timeout for querying providers on the default OTA provider list
 
     using ProviderLocationType = app::Clusters::OtaSoftwareUpdateRequestor::Structs::ProviderLocation::Type;
-    ProviderLocationType mLastUsedProvider; // Provider location used for the last query or update
+    chip::Optional<ProviderLocationType> mLastUsedProvider; // Provider location used for the last query or update
```

Comment:
nit: This is already wrapped in the chip namespace
```suggestion
   Optional<ProviderLocationType> mLastUsedProvider; // Provider location used for the last query or update
```